### PR TITLE
Add ClipBucket

### PR DESCRIPTION
**Pull request by tiritibambix**
Adding ClipBucket to the list

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/clipbucket.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: "ClipBucket is an Open Source and freely downloadable PHP script that will let you start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - All Rights Reseved
+platforms:
+  - Javascript
+  - PHP
+  - HTML
+  - CSS
+  - Shell
+  - Batchfile
```
Comment (Ki-er) [inline software/clipbucket.yml:13]: > https://github.com/MacWarrior/clipbucket-v5?tab=readme-ov-file#installation

```suggestion
```

File: software/clipbucket.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - Docker
+tags:
+  - Media Streaming - Multimedia Streaming
+  - Media Streaming - Video Streaming
+  - Photo and Video Galleries
+  - Content Management Systems (CMS)
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/clipbucket.yml:20]: Optional so can be removed

```suggestion
```

File: software/clipbucket.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - HTML
+  - CSS
+  - Shell
+  - Batchfile
+  - Docker
+tags:
+  - Media Streaming - Multimedia Streaming
+  - Media Streaming - Video Streaming
+  - Photo and Video Galleries
+  - Content Management Systems (CMS)
```
Comment (Ki-er) [inline software/clipbucket.yml:19]: Video can be under multimedia while CMS is usually for traditional web content management systems.

```suggestion
tags:
  - Media Streaming - Multimedia Streaming
  - Photo and Video Galleries
```

File: software/clipbucket.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "ClipBucket is an Open Source and freely downloadable PHP script that will let you start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
```
Comment (Ki-er) [inline software/clipbucket.yml:4]:
```suggestion
description: "Start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Some changes.

**Comment by tiritibambix** [comment]
I'm sorry I'm pretty much what we call a noob.
Why did the merge fail ? What can I do  ?

**Comment by Ki-er** [comment]
It failed due to the license `All Rights Reserved` not being present in https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/licenses-nonfree.yml. Id recommend adding it to that list with this PR.

File: software/clipbucket.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "Start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - All Rights Reseved
```
Comment (nodiscc) [inline software/clipbucket.yml:6]: I don't think this is accurate

https://github.com/MacWarrior/clipbucket-v5/blob/master/LICENSE

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**